src/GeneralRules/ReGlobalVariablesUsageRule.class.st

```diff
@@ -16,17 +16,12 @@ ReGlobalVariablesUsageRule class >> checksMethod [
 
 { #category : #running }
 ReGlobalVariablesUsageRule >> check: aMethod forCritiquesDo: aCriticBlock [
-	"Beware that ClassVariables respond true to #isGlobal. 
-	 The trick with 'ifAbsent:[Object] is basically a way not select ClassVariables (or Pools)"
 	aMethod ast allChildren
 		select: [ :node |
 			node isVariable and: [ 
-				node isGlobal and: [
+				node variable isGlobalVariable and: [
```

Comment:
we should have something like isClassBinding to distinguish globals like Smalltalk from classes or traits